src/main/java/org/fundacion/jala/converter/service/videoclasses/Converter.java

```diff
@@ -36,7 +36,7 @@ public void convertVideo(final String pathFile) {
         pathOutput = adaptPath.substring(0, (adaptPath.lastIndexOf("archive"))) + "archive\\";
         String fCommand = startFirstCommand + adaptPath + " ";
         String parameters = changeResolution() + changeFrameRate() + removeAudio();
-        String theCommand = fCommand + parameters + pathOutput + output  + "\"" + generateATumbnail() + " -y";
+        String theCommand = fCommand + parameters + pathOutput + output  + "\"" + " -y";
         System.out.println(theCommand);
         try {
```

Comment:
Logs are missing on the try - catch in this class

Reply:
Thanks. I just did it